terra-batch-libchart/templates/_wds.tpl

```diff
@@ -81,14 +81,7 @@ metadata:
   name: {{ include "app.fullname" . }}-wds-config
 data:
   {{ .Values.wds.conf_file }}: |-
-    spring:
```

Comment:
I don't think we actually need any config overrides for WDS, as the only environment-specific values are controlled by EVs (which the helm chart sets): https://github.com/DataBiosphere/terra-workspace-data-service/blob/main/service/src/main/resources/application.properties